### DISCUSSION TITLE
VAWT pitch angle - is in the direction inside the rotor or outside the rotor???

### DISCUSSION THREAD

**Discussion by victormendoza1983** [turbinesFoam/turbinesFoam#353] [General]
Hello Pete, 
In turbinesFoam for VAWT when I define the blade pitch angle in fvOptions (elemendata) this pitching correspond to the leading edge rotating inside or outside the rotor??? 

* If I am not wrong there is not a standard nomenclature for this (depends on the author/researcher)
* Can you point me in which line of the code (src) is this clarified? I am struggling to find the answer by myself.
* I attach you an image with an example for my question. For example, if I define blade pitch angle = 2 deg. These 2 degrees are implemented towards in or outside the rotor?
![2022-10-09 11-44-00](https://user-images.githubusercontent.com/97036019/194749838-ba46eb91-455c-4ad0-97c1-ad7d98350e6f.jpeg)

![2022-10-09 11-44-00](https://user-images.githubusercontent.com/97036019/194749849-d3ae33da-3b72-4d16-983a-e97e5d174f93.jpeg)

**Comment by OpenFoam-User** [comment]
@victormendoza1983 Any updates on this?
Thank you

**Comment by petebachant** [comment]
I believe the axis of rotation for pitch is pointing in the same direction as the rotor's axis of rotation, but you'd need to take a look in `crossFlowTurbineALSource.C` to see exactly how each of the actuator lines were specified. They should be created in a generic way such that they have a `spanDirection` about which pitching happens. So the process is something like:

- In the turbine object, create the geometry necessary to create individual actuator lines
- In the actuator line objects, create the elements and call `actuatorLineElement::pitch` according to the geometry
